packages/common/testing/src/mockInteractionResponseBody.json

```diff
@@ -639,6 +675,414 @@
         ]
       }
     },
+    {
```

Comment:
You no longer need this extra prescription. The changes to this file and to mockAPIResponseBody can be reverted.

Reply:
Test data has been restored.